OpenImis.ModulesV2/InsureeModule/Models/EnrollFamilyModels/Family.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenImis.ModulesV2.InsureeModule.Models.EnrollFamilyModels
+{
+    public class Family
```

Comment:
We need to create a ticket to improve the Family model to follow the model from the specification document, section 2.1.1.